spring-xd-messagebus-kafka/src/main/java/org/springframework/xd/dirt/integration/kafka/KafkaMessageBus.java

```diff
@@ -720,7 +718,7 @@ private KafkaMessageListenerContainer createMessageListenerContainer(KafkaProper
 	}
 
 	private KafkaMessageListenerContainer createMessageListenerContainer(KafkaPropertiesAccessor accessor,
-			String group, int numThreads, String topic,
+			String group, int maxConcurrency, String topic,
```

Comment:
`s/numThreads/maxConcurrency/` in the other `createMessageListenerContainer()` methods?